### PR TITLE
gh-92047: Py_GetVersion multi-digit minor version (GH-92047)

### PR DESCRIPTION
**Pull request by jebob**
Minor doc change.

### CODE REVIEW

File: Misc/NEWS.d/next/Documentation/2022-04-29-16-23-14.gh-issue-92047.oKYyuI.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+From https://docs.python.org/3.11/c-api/init.html?highlight=py_getversion#c.Py_GetVersion:
```
Comment (vstinner) [inline Misc/NEWS.d/next/Documentation/2022-04-29-16-23-14.gh-issue-92047.oKYyuI.rst]: Please remove this NEWS entry, it's not needed.

Comment (jebob) [reply Misc/NEWS.d/next/Documentation/2022-04-29-16-23-14.gh-issue-92047.oKYyuI.rst]: Removed.

**Review by kumaraditya303 — APPROVED** [review]

**Comment by vstinner** [comment]
4 required CI jobs never started. I closed/reopened the PR to schedule new CI jobs.

**Comment by vstinner** [comment]
Thanks merged. Firefox got problems with its version... 100, 3 digits :-D See https://hacks.mozilla.org/2022/02/version-100-in-chrome-and-firefox/

**Pull request merged**